### PR TITLE
test branch protection 2

### PR DESCRIPTION
**Pull request by michaelxswe**
looks good

### CODE REVIEW

**Review by michaelxia99 — APPROVED** [review]

**Review by michaelxia99 — APPROVED** [review]
i reviewed it and it looks good?

**Review by michaelxia99 — COMMENTED** [review]
doesnt look good

**Review by michaelxia99 — CHANGES_REQUESTED** [review]
nope

**Review by michaelxswe — COMMENTED** [review]
please review it everyone

File: README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ bug fix 1 is complete
 bug fix 2 is complete
 bug fix 3 is complete
 bug fix 4 is complete
-test branch protection
```
Comment (michaelxia99) [inline README.md:6]: ya this line is wrong

File: README.md [hunk 2]
```diff
@@ -3,4 +3,4 @@ bug fix 1 is complete
...
\ No newline at end of file
+test branch protection 2
```
Comment (michaelxia99) [inline README.md:6]: good solution

**Review by michaelxia99 — APPROVED** [review]
nice work

File: README.md (hunk 1 above)
Comment (michaelxswe) [reply README.md:6]: yup

File: README.md (hunk 2 above)
Comment (michaelxswe) [reply README.md:6]: thanks

**Pull request merged**